documentation/2.0/content/userguide/target_env.md

```diff
@@ -24,7 +33,73 @@ This example would apply the `k8s` target type to the discovery result, and plac
 
 If a variable file is specified on the tool's command line using the `-variable_file` argument, any injected variables will be added to that file. If no variable file is specified, injected variables will be written to the file `<output-directory>/<target_name>_variable.properties`.
 
-### The target configuration file
+### Pre-configured target environments
+
+These target environment configurations are included in the WebLogic Deploy Tooling installation.
+
+#### The WebLogic Kubernetes Operator target
+
+This target environment can be applied by providing the command-line argument `-target wko`. It will provide this additional processing:
```

Comment:
This target environment can be applied by providing -> (to change this from passive to active voice) -> You can apply this target environment by providing (if you agree, then please make this change globally)